### DISCUSSION TITLE
Map missing from homeassitant AND from the  xiaomi app

### DISCUSSION THREAD

**Discussion by poseidogithub** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#560] [Q&A] [answered]
Hello

Yesterday I installed Xiaomi Cloud Map Extractor and Xiaomi Miio. Yesterday they did worl fine, I coudl see the map in home assistant...
but today, I can not see the map on homeassitant neither on the xiaomi app (but I can control the vacuum both from homeassitatn and from the app...)  I dont know if this has to do with Xiaomi map extractor... or it's an app problem, but I'm very puzzeld.
Any hint?

**Comment by PiotrMachowski** [comment, marked as answer]
https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/issues/422

**Reply by poseidogithub** [reply]
Thanks

I wasnt sure it had to do somwthing with the threads about no map loading, as to me the most strange thing was that the app was not showing it either! 

I reseted the vacuum, (pressing power for 4 seconds, power off and on) and also added your blueprint, selected two intervals and moved the Docked interval to twice as default... lets si how this goes. thanks!
